v63004/u_inref/gtm8202.csh

```diff
@@ -0,0 +1,131 @@
+echo "#  expected to return sequences 1"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno=~0,1,~2 "a.mjl" >>& db_log.txt
+echo "#  search extract file for set variables"
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  multi region extract -SEQNO=~1,2,3,4,5 "
+echo "#  expected to return only variables a and b (not c)"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno=~1,2,3,4,5 "*" >>& db_log.txt
+echo "#  search extract file for set variables"
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  multi region extract -SEQNO=~1,2,3,4,5,6 "
+echo "#  expected to return sequences 2,3,4,5"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno=~1,2,3,4,5,6 "*" >>& db_log.txt
+echo "#  search extract file for set variables"
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  multi region extract -SEQNO=~1,2,~3"
+echo "#  expected to return sequences 2"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno=~1,2,~3 "*" >>& db_log.txt
+echo "#  search extract file for set variables"
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  multi region extract -SEQNO=~(~1,2,~3)"
+echo "#  expected to return sequences 1,3"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno="~(~1,2,~3)" "*" >>& db_log.txt
+echo "#  search extract file for set variables"
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  multi region extract -SEQNO=1,~2,3"
+echo "#  expected to return sequences 1,3"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno=1,~2,3 "*" >>& db_log.txt
+echo "#  search extract file for set variables"
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  multi region extract -SEQNO=~(1,~2,3)"
+echo "#  expected to return sequences 2"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno="~(1,~2,3)" "*" >>& db_log.txt
+echo "#  search extract file for set variables"
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  single region extract -SEQNO=1,2,3,4,5,6,7,8 TP transaction "
+echo "#  expecting a.lost to be created"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno=1,2,3,4,5,6,7,8 "a.mjl" >>& db_log.txt;
+if (  -f "a.lost" ) then
+	echo "a.lost file created (expected)"
```

Comment:
In addition to saying a.lost did get created, we need to display the contents of a.lost and Reg.mjf to ensure the output is the same every time around. So filter the contents of those two files just like you filter the contents in previous cases.